src/routes.ts

```diff
@@ -10,4 +15,5 @@ export type RootStackParamList = {
   Home: undefined;
   Details: undefined;
   StationSelect: undefined;
+  FareResults: FareResultsParam;
```

Comment:
nitpick: probably `Params` rather than `Param` - there can be multiple parameters